src/browser/base/content/zen-styles/zen-browser-ui.css

```diff
@@ -155,6 +155,9 @@
   flex-direction: row;
   position: relative;
   gap: var(--zen-element-separation);
+  :root[inDOMFullscreen] & {
```

Comment:
You realize this is used under a specific `:root:not([inDOMFullscreen='true'])` right?